examples/seismic/skew_self_adjoint/test_wavesolver_iso.py

```diff
@@ -0,0 +1,542 @@
+from scipy.special import hankel2
```

Comment:
@mloubout do we want this file to live here or rather inside our main `tests` folder?

I *think* we should stick to the current convention and move it to tests?  we could perhaps add `tests/examples/...` ? sooner or later we'll have to add subfolders to `tests` to group together tests according to some logic, and this could be a start.

I don't know -- thoughts ?

Reply:
I thought so too, but the friggin signature of that method will be loooooong